pyproject.toml

```diff
@@ -122,3 +122,12 @@ command_line = '-m pytest'
 # TODO: Cannot report relative to project path because we cannot do editable installs yet
 #source = ['python/spglib']
 source = ['spglib']
+
+[tool.ruff]
+line-length = 88
+exclude = [
+    "database",
+]
```

Comment:
By default `ruff` does not include any linting. It should include something like:
```toml
[tool.ruff.lint]
extend-select = [
    "F",           # pyflakes
    "W",           # pycodestyle-warnings
    "E",           # pycodestyle-errors
    "C90",         # mccabe
    "I",           # isort
    "N",           # pep8-naming
    "D",           # pydocstyle
    "UP",          # pyupgrade
    "YTT",         # flake8-2020
    "ANN",         # flake8-annotations
    "S",           # flake8-bandit
    "BLE",         # flake8-blind-except
    "FBT",         # flake8-boolean-trap
    "B",           # flake8-bugbear
    "A",           # flake8-builtins
    "COM",         # flake8-comma
    "C4",          # flake8-comprehensions
    "T10",         # flake8-debugger
    "EM",          # flake8-errmsg
    "FA",          # flake8-future-annotations
    "ISC",         # flake8-implicit-str-concat
    "ICN",         # flake8-import-conventions
    "G",           # flake8-logging-format
    "PIE",         # flake8-pie
    "T20",         # flake8-print
    "PYI",         # flake8-pyi
    "PT",          # flake8-pytest-style
    "Q",           # flake8-quotes
    "RSE",         # flake8-raise
    "RET",         # flake8-return
    "SLF",         # flake8-self
    "SIM",         # flake8-simplify
    "TID",         # flake8-tidy-imports
    "TCH",         # flake8-type-checking
    "ARG",         # flake8-unused-arguments
    "PTH",         # flake8-use-pathlib
    "TD",          # flake8-todos
    "ERA",         # eradicate
    "PD",          # pandas-vet
    "PGH",         # pygrep-hooks
    "PL",          # pylint
    "TRY",         # tryceratops
    "FLY",         # flynt
    "NPY",         # NumPy
    "PERF",        # perflint
    "FURB",        # refurb
    "RUF",         # Ruff-specific
]
```

Reply:
Flake8's rules seem to be enabled by default. I've added isort's rules.

Reply:
Only `E` iirc. Best to be extensive. Here's a list of rules: https://docs.astral.sh/ruff/rules/

I would encourage to keep all of them, but comment out those that are not ready to be added.